boards/stm32f3discovery/dist/flash.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "##"
+echo "## Flashing $1"
+echo "##"
+
+openocd -f "board/stm32f3discovery.cfg" \
+    -c "init" \
```

Comment:
Add `-c "tcl_port 0" -c "telnet_port 0" -c "gdb_port 0"` to avoid errors if you already have a GDB server or OpenOCD running in debug-server mode connected to some other hardware device